### ISSUE TITLE
Force Vidcoder to encode in 10 bit

### ISSUE THREAD

**Issue by 55trucker** [RandomEngy/VidCoder#509]
Dave:

what entry does one have to input into the program to force the 264 encoder to always encode in a 10bit environment? ...when one opens the *advanced* tab, what would be the option string one enters into the string *window* to force 10 bit? .......as it is right now the default is 8 bit. If I input a 10 bit file for encoding the output file is reduced to 8 bit.

**Comment by RandomEngy** [comment]
Video Encoding -> Video Codec -> H.264 10-bit (x264)

**Comment by 55trucker** [comment]
How would that be entered precisely?.....after the last displayed entry....trellis2

https://imgur.com/bNG2FuL

**Comment by RandomEngy** [comment]
You want the "Video Encoding" tab, not the "Advanced Video" tab.

**Comment by 55trucker** [comment]
Doh!

suffered a brain fart there. I have two op/sys on the computer, one is a 32 bit system while the other is the 64 bit system, the 32 bit is still running the older last version of 32 bit Vidcoder which predates the 10 bit capability. It was that op/sys I was contacting you by, when I booted up the 64 bit system the 64 bit Vidcoder displayed the 10 bit option, sorry 'bout that....

one more question....the newer 4,26 beta has altered the manner that the encoding progresses, with the older versions an incomplete mkv file that is being generated is produced as an mkv file, the 4.26 beta produces a *part* file that has to complete before it transforms into the finished mkv file. 

Is this change by design?

thx,

Steve

**Comment by RandomEngy** [comment]
Yes, this change is intentional. It's meant to reduce the amount of junk that gets created from failed or cancelled files. If you want, you can check "Keep files from failed or stopped encodes" in Global options -> Advanced.
